README.md

```diff
@@ -60,6 +61,7 @@ enter => get resource details
 C-u enter => describe resource
 h => help popup
 ? => help popup
+! => run fast shell-command
```

Comment:
the wording is confusing I think. Could you rephrase to `quick run shell-command`? I think that "run fast shell-command" implies that the command is going to run fast, not that it's easy to run it

Reply:
> going to run fast, not that it's easy to run it

Haha :laughing:, you're right. I updated, thank you